pkg/cli/etcdsnapshot/etcd_snapshot.go

```diff
@@ -69,9 +69,11 @@ func commandSetup(app *cli.Context, cfg *cmds.Server, sc *server.Config) error {
 	return nil
 }
 
-// Run is an alias for Save, retained for compatibility reasons.
+// Run was an alias for Save
+// TODO: remove in v1.27
```

Comment:
Instead of todo's in the code, can we create an issue in the repo, in the 1.27 milestone (when created) to capture this?

Reply:
I have remove the todo and created https://github.com/k3s-io/k3s/issues/6598